template-for-repository/proofs/Makefile-project-defines

```diff
@@ -29,3 +29,9 @@
 
 # Preprocessor definitions -D...
 # DEFINES =
+
+# Path to arpa executable
+# ARPA =
```

Comment:
Why comment out this definition?   Both no definition and an empty definition `ARPA = ` cause `$(ARPA)` to evaluate to an empty string.